README.md

```diff
@@ -245,7 +245,7 @@ customizable. Examples are: drag, friction, gravity, simulation timestep, non-di
 - **Agent actions**: Agents' physical actions are 2D forces for holonomic motion. Agent rotation can also be controlled through a torque action (activated by setting `agent.action.u_rot_range` at agent creation time). Agents can also be equipped with continuous or discrete communication actions.
 - **Action preprocessing**: By implementing the `process_action` function of a scenario, you can modify the agents' actions before they are passed to the simulator. This is used in `controllers` (where we provide different types of controllers to use) and `dynamics` (where we provide custom robot dynamic models).
 - **Controllers**: Controllers are components that can be appended to the neural network policy or replace it completely.  We provide a `VelocityController` which can be used to treat input actions as velocities (instead of default vmas input forces). This PID controller takes velocities and outputs the forces which are fed to the simulator. See the `vel_control` debug scenario for an example.
-- **Dynamic models**: VMAS simulates holonomic dynamics models by default. Custom dynamic constraints can be enforced in an action preprocessing step. We implement `DiffDriveDynamics`, which can be used to simulate differential drive robots. See the `diff_drive` debug scenario for an example.
+- **Dynamic models**: VMAS simulates holonomic dynamics models by default. Custom dynamic constraints can be enforced in an action preprocessing step. Implementations now include `DiffDriveDynamics` for differential drive robots and `KinematicBicycleDynamics` for kinematic bicycle model. See `diff_drive` and `kinematic_bicycle` debug scenarios for examples.
```

Comment:
Amazing!
We might also want to add it to the table of debug scenarios where we have a description and a rendering for each.
If you send me a gif of the scenario I can upload it and give you the link to use in the README

Reply:
Thanks! 
Sure. I attached the gif file. Please give me the link after you have uploaded it.

![kinematic_bicycle_model](https://github.com/proroklab/VectorizedMultiAgentSimulator/assets/83649047/7eac951c-cd4c-42ae-be0f-242acfaf6a21)

Reply:
The link is here:

https://github.com/matteobettini/vmas-media/blob/main/media/scenarios/kinematic_bicycle.gif?raw=true

Reply:
Thanks!